engine/crates/engine-v2/schema/src/walkers/definition.rs

```diff
@@ -51,15 +51,15 @@ impl<'a> DefinitionWalker<'a> {
         }
     }
 
-    pub fn interfaces(&self) -> Option<Box<dyn Iterator<Item = InterfaceWalker<'a>> + 'a>> {
+    pub fn interfaces(&self) -> Option<Box<dyn ExactSizeIterator<Item = InterfaceWalker<'a>> + 'a>> {
```

Comment:
not really necessary anymore, but kept it 🤷

Reply:
yeah it's likely going to be useful again at some point